spec/factories/works.rb

```diff
@@ -16,15 +16,15 @@
   end
 
   trait :published do
-    published_edtf { '2020-02-14' }
+    published_edtf { EDTF.parse('2020-02-14') }
```

Comment:
Can you say more about why changes to this file are needed? I thought these strings were crafted to be EDTF-compliant.

Reply:
Because published_edtf= should always receive a date/interval object, never a string.

Reply:
OK